_posts/2023-01-26-rust-32.0.0.md

```diff
@@ -0,0 +1,193 @@
+* **Improved Generics**: Improved generics allow writing code generic over all arrays, or all arrays with the same layout
+* **Downcast Macros**: Various [helper](https://docs.rs/arrow/32.0.0/arrow/macro.downcast_primitive_array.html) [macros](https://docs.rs/arrow/32.0.0/arrow/macro.downcast_dictionary_array.html) are now available to simplify dynamic dispatch to statically typed implementations.
+
+# Parquet
+
+[Apache Parquet](https://parquet.apache.org/) is an open source, column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk. The Apache Parquet implementation in Rust is one of the [fastest and most sophisticated ](https://arrow.apache.org/blog/2022/12/26/querying-parquet-with-millisecond-latency/) open source implementations available.
+
+
+**Major Highlights**
+
+* **Arbitrarily Nested Schema**: Arbitrarily nested schemas can be read to and written from arrow, as described in the series of [blog posts](https://arrow.apache.org/blog/2022/10/05/arrow-parquet-encoding-part-1/) on the topic.
+* **Predicate Pushdown**: The [arrow reader](https://docs.rs/parquet/32.0.0/parquet/arrow/index.html) now supports advanced predicate pushdown, including late materialization, as described in a [the blog posts](https://arrow.apache.org/blog/2022/12/26/querying-parquet-with-millisecond-latency/). See [RowSelection](https://docs.rs/parquet/32.0.0/parquet/arrow/arrow_reader/struct.RowSelection.html) for the code.
+* **Bloom Filter Support**: Support for both [reading and writing bloom filters](https://docs.rs/parquet/32.0.0/parquet/bloom_filter/index.html) has been added.
+* **CLI Tools**: additional [CLI tools](https://github.com/apache/arrow-rs/tree/master/parquet/src/bin) have been added to introspect and manipulate parquet data.
+
+# Object Store
+
+Modern analytic workloads increasingly make use of blob storage facilities, such as S3, to store large volumes of queryable data. A native Rust object storage implementation that works well with the Rust Ecosystem and general and the Arrow IO abstractions, is an important building block for many applications.
+The [object_store](https://docs.rs/object_store/latest/object_store/) crate was [donated to the Apache Arrow project in July 2022](https://www.influxdata.com/blog/rust-object-store-donation/) to fill this need, and while it follows a separate release schedule than the `arrow` and `parquet` crates, it forms an integral part of the overarching Arrow IO story.
+
+Recent improvements include the following:
+
+**Major Highlights**
+
+* **Streaming Upload**: Multipart uploads are now supported.
+* **Minimised dependency footprint**: Upstream SDKs are no longer used, improving consistency and reducing dependencies.
+* **HTTP / WebDAV Support**: Applications can access arbitrary HTTP servers supporting `WebDAV`.
```

Comment:
```suggestion
* **HTTP / WebDAV Support**: Applications can read from arbitrary HTTP servers, with mutation and listing supported on WebDAV-compatible endpoints.
```
To make it clearer that you can read from arbitrary HTTP servers, you only need WebDAV if you want to mutate